src/transaction/ada.ts

```diff
@@ -0,0 +1,83 @@
+import axios from 'axios';
+import CardanoWasm from '@emurgo/cardano-serialization-lib-nodejs';
+
+import { TransferAda, AdaUTxo } from '../model';
+
+/**
+ * Prepare a signed ADA transaction with the private key locally. Nothing is broadcasted to the blockchain.
+ * @param body content of the transaction to prepare.
+ * @param utxos array of unspend transaction output.
+ * @param slotNo a number of slot to calculate time-to-live of the transaction
+ * @returns raw transaction data in hex, to be broadcasted to blockchain.
+ */
+export const prepareADATransaction = async (body: TransferAda, utxos: AdaUTxo[], slotNo: number) => {
```

Comment:
instead of utxos in the parameter, please call method cardanoGetUtxos from the API. Function will be much simpler for the end user to integrate

Reply:
I have fixed this. Please check it again. Thanks